### PR TITLE
Centralize linting at one place for keeping all repo in sync

### PR DESCRIPTION
**Pull request by MichelML**
The basic idea is that when you install this package it installs `tslint.json` and `tsconfig.lint.json` at the root of your repo

You can use `node node_modules/tsjs/scripts/lintfix/` to use your own `tsconfig.json` for linting.  

You can also use `node node_modules/tsjs/scripts/lintfix/ --tsconfig tsconfig.lint.json` which would fix all your js and ts files in your repo except in dist target and node_modules.   

The latter would be added in all repo under `npm run lintfix`  

`tslint.json` and tsconfig.lint.json` would go in `.hgignore`.

This is an idea, open for discussion.

### CODE REVIEW

File: scripts/lintfix/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const argv = require('yargs').argv;
+const sh = require('shelljs');
+const tslintConfig = 'tslint.json';
+const tsConfig = argv.tsconfig || 'tsconfig.json';
+
+sh.exec(`node node_modules/tslint/bin/tslint --project ${tsConfig} --config ${tslintConfig} --fix`, () => {
```
Comment (GermainBergeron) [inline scripts/lintfix/index.js]: you can probably use `tslint --project ${tsConfig} --config ${tslintConfig} --fix` instead (without the node node_modules/tslint/bin/)

File: scripts/lintinstall/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const tslintConfig = require(`${__dirname}/../../tslint.json`);
+const tsConfigLint = require(`${__dirname}/../../tsconfig.lint.json`);
```
Comment (GermainBergeron) [inline scripts/lintinstall/index.js]: you should probably use path.join(__dirname, '..', '..', 'tslint.json') to get something working in every os

File: tslint.test.json [hunk 3]
```diff
@@ -0,0 +1,347 @@
+{
```
Comment (GermainBergeron) [inline tslint.test.json]: remove, it's a copy of the other file

**Review by GermainBergeron — COMMENTED** [review]
I'm not sure that we should copy the tslint.json & tsconfig.lint.json in the other repo. It would be best to use it directly from node_modules I think

**Review by rousselm4 — APPROVED** [review]

**Comment by MichelML** [comment]
@GermainBergeron for `I'm not sure that we should copy the tslint.json & tsconfig.lint.json in the other repo. It would be best to use it directly from node_modules I think` -- this would not be standard, the tooling and tslint itself usually expect a tslint.json at the project root...I think it would be simpler that way and would require less changes in other repositories.

File: LICENSE.md [hunk 4]
```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2017 Airbnb
```
Comment (GermainBergeron) [inline LICENSE.md]: Should we just put Coveo instead of Airbnb like in other repos?

**Review by GermainBergeron — APPROVED** [review]

File: LICENSE.md (hunk 4 above)
Comment (MichelML) [reply LICENSE.md]: I don't know if we can actually...i'm not so familiar with the intricacies of licensing

**Review by stephr0y — APPROVED** [review]

**Pull request merged**